corehq/apps/reports/commtrack/standard.py

```diff
@@ -156,10 +155,9 @@ def get_prod_data(self):
             pagination=self.pagination,
         )
         product_grouping = {}
+        domain = Domain.get_by_name(self.domain)
         for ledger_value in ledger_values:
-            consumption_helper = get_consumption_helper_from_ledger_value(
-                Domain.get_by_name(self.domain), ledger_value
-            )
+            consumption_helper = get_consumption_helper_from_ledger_value(domain, ledger_value)
             status = consumption_helper.get_stock_category()
             if ledger_value.entry_id in product_grouping:
```

Comment:
Unrelated to your changes, but will this part of the if statement ever get hit? I'm not seeing where `product_grouping` gets set

Reply:
yes, this whole if statement is acting like a `defaultdict`. The `product_grouping` gets set below.